.python-version

```diff
@@ -0,0 +1 @@
+3.7.7
```

Comment:
This file got created on my machine at some point as well but I noticed it has never been pushed; does anyone know where it comes from, and whether we need it?

Reply:
I also had it at some point but now I don't see it!